Makefile

```diff
@@ -110,11 +122,45 @@ clear-cache-prod: install production.ini
 c2corg_ui/closure/%.py: $(CLOSURE_LIBRARY_PATH)/closure/bin/build/%.py
 	cp $< $@
 
-c2corg_ui/locale/c2corg_ui-client.pot: $(APP_HTML_FILES) $(APP_PARTIAL_FILES) $(APP_JS_FILES)
+# i18n and Transifex tools
+
+# if .transifexrc does not exist yet, create it for read only access (with fake user c2c)
+$(HOME)/.transifexrc:
+	echo "[https://www.transifex.com]" > $@
+	echo "hostname = https://www.transifex.com" >> $@
+	echo "username = c2c" >> $@
+	echo "password = c2cc2c" >> $@
```

Comment:
This is a dummy user (used by geomapfish too by the way) which only had read access.
The real user with write access is "c2corg" and its password is travis-encrypted. See https://github.com/c2corg/v6_ui/blob/fe1080917f558bbdcb4150af99c63a08825bfea6/.travis.yml#L26-L39